### PR TITLE
Set06| Add Node Documentation for...

### PR DESCRIPTION
**Pull request by johnpierson**
### Purpose
Add node documentation for:
View Disciplines
Visual Styles
ScheduleOnSheet.ScheduleView
Sheet.Schedules
View.AddFilter
View.FilterOverrides
View.GetCategoryOverrides
View.IsCategoryHidden
View.SetDiscipline
Visual Styles
View.SetFilterOverrides
ParameterFilterElement.ByRules

### Declarations

Check these if you believe they are true

- [x] The code base is in a better state after this PR
- [x] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [x] The level of testing this PR includes is appropriate
- [x] User facing strings, if any, are extracted into `*.resx` files
- [x] Snapshot of UI changes, if any.

### Reviewers
@achintyabhat 

### FYIs
@Amoursol

### CODE REVIEW

File: doc/distrib/NodeHelpFiles/_review/Revit.Elements.ScheduleOnSheet.ScheduleView.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+## In Depth
+`ScheduleOnSheet.ScheduleView` returns the owner schedule view used to generate the instance on the sheet.
```
Comment (achintyabhat) [inline doc/distrib/NodeHelpFiles/_review/Revit.Elements.ScheduleOnSheet.ScheduleView.md:2]: For readability and clarity, suggest rephrasing - "The ScheduleOnSheet.ScheduleView node in Dynamo for Revit is used to retrieve the original schedule view that was used to create the specific schedule instance on a sheet."

File: doc/distrib/NodeHelpFiles/_review/Revit.Elements.ScheduleOnSheet.ScheduleView_img.jpg
Comment (achintyabhat) [inline doc/distrib/NodeHelpFiles/_review/Revit.Elements.ScheduleOnSheet.ScheduleView_img.jpg:1]: Image is not very clear to read. Like how this example is giving additional tip with the filtering for revision schedules.  As this example is similar to Sheet.Schedules node, suggest keeping this example for the other node. Maybe keep this example crisp and focused on this node. Even something like this could work. 
image

File: doc/distrib/NodeHelpFiles/_review/Revit.Elements.ScheduleOnSheet.ScheduleView.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+
+In the example below, the schedule instances are returned for the selected sheet. The revision schedules are then filtered out with `List.FilterByBoolMask`. Additionally, the owner view for the schedule instance is retrieved with `ScheduleOnSheet.ScheduleView`.
```
Comment (achintyabhat) [inline doc/distrib/NodeHelpFiles/_review/Revit.Elements.ScheduleOnSheet.ScheduleView.md:4]: If the example is changed, note to update this section of the In Depth description.

File: doc/distrib/NodeHelpFiles/_review/Revit.Elements.Views.View.AddFilter.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+## In Depth
+`View.AddFilter` adds the given filter to the given view. If a view has a view template applied, `View.AddFilter` will add the filter to the view's view template.
+
```
Comment (achintyabhat) [inline doc/distrib/NodeHelpFiles/_review/Revit.Elements.Views.View.AddFilter.md:3]: This sentence has too many "Views" and my brain can't handle (lol).  Maybe rephrase as "View.AddFilter adds the specified filter to the given view. If the view has a view template applied, this node will add the filter to it's view template."

File: doc/distrib/NodeHelpFiles/_review/Revit.Elements.Views.View.AddFilter.md
Comment (achintyabhat) [inline doc/distrib/NodeHelpFiles/_review/Revit.Elements.Views.View.AddFilter.md:1]: Love the example for this node!!!

File: doc/distrib/NodeHelpFiles/_review/Revit.Elements.Views.View.FilterOverrides.md [hunk 4]
```diff
@@ -0,0 +1,8 @@
+## In Depth
+`View.FilterOverrides` provides the settings for the given view filter `(ParameterFilterElement)`.
+
+In the example below, the view filters are collected from the current view and the filter overrides are returned.
```
Comment (achintyabhat) [inline doc/distrib/NodeHelpFiles/_review/Revit.Elements.Views.View.FilterOverrides.md:4]: Note to update the example description to align with example file and image

File: doc/distrib/NodeHelpFiles/_review/Revit.Elements.Views.View.FilterOverrides_img.jpg
Comment (achintyabhat) [inline doc/distrib/NodeHelpFiles/_review/Revit.Elements.Views.View.FilterOverrides_img.jpg:1]: Note to update image of the updated dyn file.

File: doc/distrib/NodeHelpFiles/_review/Revit.Elements.Views.View.GetCategoryOverrides.dyn
Comment (achintyabhat) [inline doc/distrib/NodeHelpFiles/_review/Revit.Elements.Views.View.GetCategoryOverrides.dyn:1]: Depending on user's active view, this might return just empty lists- might be confusing for new users. 
image

File: doc/distrib/NodeHelpFiles/_review/Revit.Elements.Views.View.FilterOverrides.dyn
Comment (achintyabhat) [inline doc/distrib/NodeHelpFiles/_review/Revit.Elements.Views.View.FilterOverrides.dyn:1]: Depending on user's active view, this might return just empty lists- might be confusing for new users. 
image

File: doc/distrib/NodeHelpFiles/_review/Revit.Elements.Views.View.GetCategoryOverrides.dyn
Comment (achintyabhat) [inline doc/distrib/NodeHelpFiles/_review/Revit.Elements.Views.View.GetCategoryOverrides.dyn:1]: Maybe it is just me. Document.ActiveView node is throwing an error for me. I am in the same project and view, other examples with the same node work fine!! Will ask someone else to try as well.
Screenshot 2024-09-24 225929

File: doc/distrib/NodeHelpFiles/_review/Revit.Elements.Views.View.IsCategoryHidden.md [hunk 5]
```diff
@@ -0,0 +1,8 @@
+## In Depth
+`View.IsCategoryHidden` returns a true or false value to indicate if the category has been hidden in the given view. Note, if a view has a view template assigned, the view template must be used.
+
+In the example below, the active view's view template from the current document is retrieved. `View.IsCategoryHidden` reports if the wall category is hidden.
```
Comment (achintyabhat) [inline doc/distrib/NodeHelpFiles/_review/Revit.Elements.Views.View.IsCategoryHidden.md:4]: For readability, suggest rephrasing this as "In the example below, the view template of the active view from the current document is retrieved...." Sorry for being picky! Active view's view template is a hard to read.

File: doc/distrib/NodeHelpFiles/_review/Revit.Elements.Views.View.AddFilter.dyn
Comment (achintyabhat) [inline doc/distrib/NodeHelpFiles/_review/Revit.Elements.Views.View.AddFilter.dyn:1]: Note to update the dyn file in alignment with the comment below.

File: doc/distrib/NodeHelpFiles/_review/Revit.Elements.Views.View.AddFilter_img.jpg
Comment (achintyabhat) [inline doc/distrib/NodeHelpFiles/_review/Revit.Elements.Views.View.AddFilter_img.jpg:1]: Note to update the image file ito align with the comment above.

**Review by achintyabhat — CHANGES_REQUESTED** [review]
Thanks John for all the amazing examples on this set, very handy for users! Just some minor comments on the description for better readability. Also suggest splitting some of the examples into smaller chunks , instead of repeating the example file for multiple nodes.

**Comment by johnpierson** [comment]
I believe all comments are now addressed @achintyabhat ! Thanks for that! All made sense to me and improved the readability for sure.

**Review by achintyabhat — APPROVED** [review]
Thanks @johnpierson for all the changes. Looks great!
